examples/sagaMonitor/modules/DescriptorFormatter.js

```diff
@@ -0,0 +1,56 @@
+import { is } from 'redux-saga/utils'
+import Formatter from './Formatter'
+import { CANCELLED, PENDING, REJECTED, RESOLVED } from './constants'
+
+const DEFAULT_STYLE = 'color: black'
+const LABEL_STYLE = 'font-weight: bold'
+const EFFECT_TYPE_STYLE = 'color: blue'
+const ERROR_STYLE = 'color: red'
+const CANCEL_STYLE = 'color: #ccc'
+
+export default class DescriptorFormatter extends Formatter {
```

Comment:
are we going to reuse `Formatter`? imho implementation with inheritance a bit confusing.

Reply:
The `Formatter` is a universal formatter which is adapted from the old code. And `DescriptorFormatter` is a special formatter that knows the structure of effect descriptors. So conceptually, `DescriptorFormatter` is a special kind of `Formatter`, so I use inheritance.